streamer/src/nonblocking/quic.rs

```diff
@@ -1712,7 +1720,6 @@ pub mod test {
         // Test that more writes to the stream will fail (i.e. the stream is no longer writable
         // after the timeouts)
         assert!(s1.write_all(&[0u8]).await.is_err());
-        assert!(s1.finish().await.is_err());
```

Comment:
why remove this?

Reply:
finish is made a sync function. it is no longer returning error in this in 0.11.x under this case.